synapse/http/client.py

```diff
@@ -353,18 +376,26 @@ async def request(self, method, uri, data=None, headers=None):
                 set_tag("error_reason", e.args[0])
                 raise
 
-    async def post_urlencoded_get_json(self, uri, args={}, headers=None):
+    async def post_urlencoded_get_json(
+        self,
+        uri: str,
+        args: Dict[str, Union[str, List[str]]] = {},
```

Comment:
Is this meant to be the `QueryParams` type? I think not since it is used differently than `get_raw` does, but wanted to double check!

Reply:
I hadn't really thought about it tbh. It's worth noting that they aren't actually query params, and as you say it's used differently. I just copied the old type def from the docstring here.

Reply:
Seems reasonable to leave it this way then. 👍